catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -141,6 +141,29 @@ const WorkDetails = ({
   const locationOfWork = work.notes.find(
     note => note.noteType.id === 'location-of-original'
   );
+  const arrangementNote = work.notes.find(
+    note => note.noteType.id === 'arrangement-note'
+  );
+  const biographicalNote = work.notes.find(
+    note => note.noteType.id === 'biographical-note'
+  );
+  const relatedMaterial = work.notes.find(
+    note => note.noteType.id === 'related-material'
+  );
+  const acquisitionNote = work.notes.find(
+    note => note.noteType.id === 'acquisition-note'
+  );
+
+  const orderedNotes = [
+    arrangementNote,
+    acquisitionNote,
+    biographicalNote,
+    relatedMaterial,
+  ].filter(Boolean);
+
+  const remainingNotes = work.notes.filter(note => {
```

Comment:
Feels like there's probably some clever way of doing this (and the above) without so many list searches but it's so easy to read this way that I think it's fine if there aren't too many note fields